### PR TITLE
Qu auto/sorqa 171 Bulk mode for Activate/deactivate user accounts[SORDEV-5964]

### PR DESCRIPTION
**Pull request by mk-sgent**

### CODE REVIEW

**Comment by mk-sgent** [comment]
SORQA-171

Jenkins jobs:
https://jenkins.sormas.netzlink.com/job/sormas-RunE2eTests-branch/725/allure/#suites/bd663fa712ccafbe2d65d1dd364e02f0/a13137754208214a/

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/users/CreateNewUserSteps.java [hunk 1]
```diff
@@ -56,6 +62,65 @@ public CreateNewUserSteps(
     this.webDriverHelpers = webDriverHelpers;
     this.baseSteps = baseSteps;
 
+    When(
+        "^I pick and count amount a users that was created on the same hour$",
+        () -> {
+            String userDataPartial = user.getUserName().substring(0, 17);
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/users/CreateNewUserSteps.java:68]: use the full user uuid, is more stable and precise

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/users/CreateNewUserSteps.java [hunk 2]
```diff
@@ -46,6 +51,7 @@ public class CreateNewUserSteps implements En {
   public static String userName;
   public static String userPass;
   private final BaseSteps baseSteps;
+  private static Integer amountOfRecords;
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/users/CreateNewUserSteps.java:54]: declare it as int

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/users/CreateNewUserSteps.java [hunk 3]
```diff
@@ -56,6 +62,65 @@ public CreateNewUserSteps(
...
+        "^I pick and count amount a users that was created on the same hour$",
+        () -> {
+            String userDataPartial = user.getUserName().substring(0, 17);
+          webDriverHelpers.fillInWebElement(USER_INPUT_SEARCH, userDataPartial);
+          TimeUnit.SECONDS.sleep(5); // wait for page loaded
+          String amountOfUsers = webDriverHelpers.getTextFromWebElement(AMOUNT_OF_CHOSEN_USERS);
+          amountOfRecords = Integer.parseInt(amountOfUsers);
+        });
+
+    When("I Count amount of user", () -> {});
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/users/CreateNewUserSteps.java:75]: why is this method empty?

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/users/CreateNewUserSteps.java [hunk 4]
```diff
@@ -56,6 +62,65 @@ public CreateNewUserSteps(
...
+        });
+
+    And(
+        "I check that all Users are changed Active field value to opposite",
+        () -> {
+          String amountOfCheckboxes =
+              webDriverHelpers.getTextFromWebElement(AMOUNT_ACTIVE_INACTIVE_USERS);
+          Integer optionsRecords = Integer.parseInt(amountOfCheckboxes);
+          softly.assertEquals(
+              amountOfRecords, optionsRecords, "Not all Active fields value are changed");
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/users/CreateNewUserSteps.java:121]: you need to add also softly.assertAll to have the check validated

**Review by rdutu-vg — CHANGES_REQUESTED** [review]

**Comment by rdutu-vg** [comment]
address pr comments and re-run the test

**Review by rdutu-vg — APPROVED** [review]

**Pull request merged**